crossdock/otel-agent-config.yml

```diff
@@ -4,15 +4,19 @@ receivers:
       host_endpoint: localhost:5778
       fetch_endpoint: jaeger-collector:14250
     protocols:
+      grpc:
+        disabled: true
       thrift_compact:
       thrift_binary:
 
 exporters:
+  jaeger_cassandra:
+    disabled: true
   jaeger:
     endpoint: jaeger-collector:14250
+    wait_for_ready: true
 
 service:
   pipelines:
     traces:
-      receivers: [jaeger]
```

Comment:
Why has jaeger receiver been dropped?

Are we also going to support an agent config using otel protocol receiver and exporter config? For example when using an OTel tracer with otel protocol exporter - it might be good to collect via otel agent (no jaeger receiver/exporter) and talk otel protocol to the backend jaeger collector, i.e. so no jaeger protocol used.

Reply:
> Why has jaeger receiver been dropped?

Jaeger receiver is enabled by default, hence we don't have to specify it explicitly. In this config file we just disabled the grpc endpoint - to better align with legacy agent capabilities. Once we create agent OTEL build this config file will change bc the agent will use a different hardcoded default config https://github.com/jaegertracing/jaeger/issues/2204 

> Are we also going to support an agent config using otel protocol receiver and exporter config? For example when using an OTel tracer with otel protocol exporter - it might be good to collect via otel agent (no jaeger receiver/exporter) and talk otel protocol to the backend jaeger collector, i.e. so no jaeger protocol used.

This is a separate discussion but I agree, we should be enabling OTEL endpoints by default. https://github.com/jaegertracing/jaeger/issues/2214

Reply:
See #2214 has been created to deal with the second point separately.
> Why has jaeger receiver been dropped?

Assume this is because it is in default config.